lib/expand.js

```diff
@@ -590,7 +590,7 @@ function _expandValue({activeCtx, activeProperty, value}) {
 
   const rval = {};
 
-  if(type !== null) {
+  if(type && ['@id', '@vocab'].indexOf(type) === -1) {
```

Comment:
Should use `! .includes` instead of `indexOf` check.

Reply:
With two items is it better to just expand out the two checks?

Reply:
Perhaps, but it makes it easier to alter in the future; I'll update to use `!.includes`.

Reply:
Should https://github.com/gkellogg/jsonld.js/blob/no-expand-native-as-object/lib/expand.js#L421-L423 and https://github.com/gkellogg/jsonld.js/blob/no-expand-native-as-object/lib/expand.js#L604  use this convention too?

Reply:
I think all `indexOf() ===/!== -1` uses can be switched over to `includes()`.